cmd/engine/engine.go

```diff
@@ -394,10 +405,13 @@ func doCmd(_ *cobra.Command, _ []string) (err error) {
 	log.Infof("Starting gRPC endpoint on :%d", grpcPort)
 
 	// Automatically start the discovery, if we have this flag enabled
-	if viper.GetBool(DiscoveryAutoStartFlag) {
+	getBool := viper.GetBool(DiscoveryAutoStartFlag)
```

Comment:
what's the point of this bool?

Reply:
Oh, it was for debugging. I change it back